src/http-server.ts

```diff
@@ -1,5 +1,17 @@
+import { Server } from 'http';
+import { Socket } from 'net';
+import type { Express } from 'express';
+
 export class HttpServer {
-  constructor(app, keepAliveTimeoutMillis, headersTimeoutMillis) {
+  private app: Express;
+  private tcpSockets: { [socketId: number]: Socket };
+  private tcpSocketId: number;
```

Comment:
NIT: can we name this something more descriptive like `nextTcpSocketId`?

Reply:
does adding the `next` provide some indication that it changes that `tcpSocketId` doesn't?